### PR TITLE
Add Services Section

### PR DESCRIPTION
**Pull request by cloudymind**
Issue #13 

*Includes small cleanup to point to arun-gupta repo

### CODE REVIEW

File: deployment-concepts/services.adoc [hunk 1]
```diff
@@ -1 +1,149 @@
...
+	spec:
+	  replicas: 3 
+	  template:
+	    metadata:
+	      labels:
+	        app: demo-app 
+	    spec:
+	      containers:
+	      - name: echoheaders 
+	        image: gcr.io/google_containers/echoserver:1.4
```
Comment (arun-gupta) [inline deployment-concepts/services.adoc]: Can we use images hosted on ECR or Docker Hub instead?

File: templates/echo.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+spec:
+  replicas: 3 
+  template:
+    metadata:
+      labels:
+        app: demo-app 
+    spec:
+      containers:
+      - name: echoheaders 
+        image: gcr.io/google_containers/echoserver:1.4
```
Comment (arun-gupta) [inline templates/echo.yaml]: Ditto here

Comment (cloudymind) [reply templates/echo.yaml]: Yeah, I couldn't find an official in dockerhub so went with this one. Let me poke around and try to find one.

Comment (christopherhein) [reply templates/echo.yaml]: could you just pull the latest and tag it and push to hub? If you want I can make you a repo under `awspartners` for Docker Hub?

File: deployment-concepts/services.adoc [hunk 3]
```diff
@@ -1 +1,149 @@
...
+If you would like to expose a service to an external IP, ServiceTypes are used to determine the type of service. Those ServiceTypes are:
+
+	ClusterIP: Service exposed on a internal cluster IP.
+	NodePort: Service exposed on each Node's IP at a defined port.
+	LoadBalancer: Service exposed externally using a cloud based load balancer. 
+	ExternalName: Service is attached to the externalName field. It is mapped to a CNAME with the value.
+
+Below we will provision a Load Balancer and expose your services, add a type field of LoadBalancer. 
+
+First deploy an app. In this example, we will create an echo app that responds with http headers from an Elastic Load Balancer:
```
Comment (christopherhein) [inline deployment-concepts/services.adoc]: nomenclature should that be `HTTP`  rather than `http`? and maybe `First, ...`

**Comment by arun-gupta** [comment]
Merging it for now, let's take care of this during testing.

**Pull request merged**